purchase_advance_payment/tests/test_purchase_advance_payment.py

```diff
@@ -121,6 +130,13 @@ def setUpClass(cls):
             }
         )
 
+    @classmethod
+    def tearDownClass(cls):
+        # inactive euro currency if active through test
+        if cls.active_euro:
+            cls.currency_euro.active = False
```

Comment:
This is pointless AFAICS.
There's no need to manually deactivate EUR, the transaction will be rolledback.

Or am I missing something here? What lead you to add this code?

Reply:
Removed, Thanks